src/wallet/wallet.h

```diff
@@ -78,7 +78,8 @@ enum WalletFeature
     FEATURE_WALLETCRYPT = 40000, // wallet encryption
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
-    FEATURE_LATEST = 60000
+    FEATURE_HD = 129900, // Hierarchical key derivation after BIP32 (HD Wallet)
```

Comment:
It is the client version: https://dev.visucore.com/bitcoin/doxygen/class_c_wallet.html#a3e37adcf1b2ad9fc02ae3e8da7c692c9